tools/README.md

```diff
@@ -108,3 +108,48 @@ def allocate_memory():
 
 max_memory_usage: float = mmc.memory_data[MemoryType.SYSTEM]
 ```
+
+## Visualization of Weight Compression results
```

Comment:
What you think about have individual readme file for each tool?

Reply:
Then we need individual folder... probably too much for tools